dashboard/app/models/artist.rb

```diff
@@ -264,4 +275,44 @@ def k1_blocks
       <block type="simple_jump_left"></block>
     XML
   end
+
+  def strip_block(block)
+    stripped_block = block.dup
+    stripped_block.xpath(STRIPPED_NODES_XPATH).remove
+    stripped_block['type'] = stripped_block['type'].chomp '_dropdown'
```

Comment:
Oh, that's true. I was only considering the opposite case, where a dropdown is available in the toolbox but the solution is written using a constant. I found quite a few levels like that, and I think it would be more annoying than useful to add the constant block every time they're edited.

Updated to allow a *_dropdown block in the toolbox to match a *_constant block in the solution, but not vice versa. LMK if you think that's ok.